docs/enterprise-solutions/configuration/infrastructure-configuration/control-other-cline-features/yolo-mode.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Yolo Mode
+description: Configure Yolo Mode settings for enterprise environments
+---
+
+## Overview
+
+Yolo Mode enables Cline to operate with increased automation while maintaining enterprise security and compliance standards. When enabled, Cline can execute approved operations with minimal user confirmation, accelerating development workflows.
+
+## Configuration
+
+### Enabling Yolo Mode
+
+Yolo Mode can be configured at the organization level through your enterprise dashboard:
+
+1. Navigate to your enterprise configuration settings
+2. Access the "Advanced Features" section  
+3. Toggle Yolo Mode for your organization
+4. Configure specific automation rules and permissions
+
+### Permission Levels
+
+Yolo Mode operates with different permission levels:
+
+- **Limited**: Basic automation for safe operations
+- **Standard**: Expanded automation for common development tasks
+- **Advanced**: Maximum automation with enterprise oversight
```

Comment:
I'm not familiar with this levels. Is this coming soon? Otherwise would recommend removing this section